config/configgrpc/configgrpc.go

```diff
@@ -90,12 +95,27 @@ type ClientConfig struct {
 	Auth *configauth.Authentication `mapstructure:"auth"`
 }
 
+// NewDefaultClientConfig() creates a new ClientConfig with any default values set
+func NewDefaultClientConfig() *ClientConfig {
+	return &ClientConfig{
```

Comment:
This needs to use `configtls.NewDefaultClientConfig once it exists.

Reply:
Concerning this too, can i get a link to the issue that will resolve this so I can work on it

Reply:
I have prepared a branch that will set default functions for `configtls`, but an issue has to first be created. Or should i go ahead and create the issue, then link a PR for the issue? @TylerHelmuth